packages/lodestar/src/api/rpc/api/beacon/beacon.ts

```diff
@@ -31,9 +31,13 @@ export class BeaconApi implements IBeaconApi {
     return Buffer.from(`lodestar-${process.env.npm_package_version}`, 'utf-8');
   }
 
-  public async getFork(): Promise<Fork> {
+  public async getFork(): Promise<{fork: Fork; chain_id: uint64}> {
     const state: BeaconState = await this.db.state.getLatest();
-    return state.fork;
+    const networkId: uint64 = await this.chain.networkId;
```

Comment:
could you create `{fork: BeaconState, chainId: number64}` in impl directory and use it from both rest and rpc so we don't have duplicate code